server/build/Dockerfile

```
@@ -39,7 +39,7 @@ HEALTHCHECK --interval=30s --timeout=10s \
   CMD curl -f http://localhost:8065/api/v4/system/ping || exit 1
 
 # Configure entrypoint and command
-COPY entrypoint.sh /
+COPY --chown=mattemrost:mattermost --chmod=665 entrypoint.sh /
```

Alternatively, could we specify the entrypoint as `/bin/bash entrypoint.sh`?

---

There is also a typo in UID here 

```suggestion
COPY --chown=mattermost:mattermost --chmod=665 entrypoint.sh /
```